R/git.R

```diff
@@ -14,6 +14,10 @@ git_uncommitted <- function(path = ".") {
   any(st != 0)
 }
 
+git_committed <- function(path = ".") {
```

Comment:
Given that this function is not just the negation of `git_committed()` I think it's confusing.